src/DependencyInjection/Configuration.php

```diff
@@ -39,13 +39,48 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->arrayPrototype()
                             ->children()
                                 ->scalarNode('url')->info('URL of the hub\'s publish endpoint')->example('https://demo.mercure.rocks/.well-known/mercure')->end()
-                                ->scalarNode('jwt')->info('JSON Web Token to use to publish to this hub.')->end()
-                                ->scalarNode('jwt_provider')->info('The ID of a service to call to generate the JSON Web Token.')->end()
+                                ->arrayNode('token')
```

Comment:
Can't we keep `jwt` and deprecate passing a string?

Reply:
Hm, we can actually use `jwt`, and *not* deprecate passing a string, `jwt: "foobar"` can be considered the same as `jwt: {"token": "foobar"}`

Reply:
I think we should go this way, most people ( those not using costume token providers ), don't have to change any configuration to migration :) 

```yaml
mercure:
  hubs:
    url: https://void.tn/.well-known/mercure
    jwt: foo.bar.baz
```
is same as:

```yaml
mercure:
  hubs:
    url: https://void.tn/.well-known/mercure
    jwt:
      value: foo.bar.baz
```

Reply:
perfect